### ISSUE TITLE
Sync keeps going for folders with dots in the end

### ISSUE THREAD

**Issue by RussellDash332** [Beebeeoii/lominus#89]
**Describe the bug**
Similar to #88 but this time it keeps downloading instead of an error. This is because the folder created omits the dot at the back so it keeps assuming the files aren't there yet.

**Logfile snapshot**
```
[2022-09-21 11:37:38] DEBUG - downloading - EC1101E/Exam Instructions etc./EC1101E Midterm Exam Instructions.pdf [0001-01-01 00:00:00 +0000 UTC vs 2022-09-16 18:06:01.36 +0800 +08] (D:/jiawei/projects/lominus/internal/cron/cron.go:github.com/beebeeoii/lominus/internal/cron.createJob.func1:224)
[2022-09-21 11:37:38] INFO - job completed: 2022-09-21T11:37:38+08:00 (D:/jiawei/projects/lominus/internal/cron/cron.go:github.com/beebeeoii/lominus/internal/cron.createJob.func1:275)
```

**OS**
Windows 10

**Comment by RussellDash332** [comment]
This seems to be a Windows-only issue so far. Not sure for Mac, but I think this doesn't happen on Linux because it doesn't auto-truncate directories ending with dots. The ad-hoc solution is to use bash to force-create the directory with `mkdir`.

**Comment by Beebeeoii** [comment]
issue should be resolved by 0a74143b1294463124a9301952f3f8f8dda55056

**Comment by RussellDash332** [comment]
> issue should be resolved by 0a74143b1294463124a9301952f3f8f8dda55056

Great! Let me test with a sandbox course on Canvas later

**Comment by Beebeeoii** [comment]
resolved with #96
